spec/models/two_factors_token_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe TwoFactorsToken, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
It would be great to have tests that confirm:
1. if you choose to remember your device, it remembers it.  
2. if you don't choose to remember, you are prompted for 2FA on the next login
3. if you travel 31 days in the future (see: https://api.rubyonrails.org/v5.2.4.4/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel_to) that you are re-prompted for your code.